packages/ui/src/components/va-pagination/VaPagination.demo.vue

```diff
@@ -1,216 +1,376 @@
 <template>
-  <VbDemo>
+  <VbDemo class="va-pagination-demo">
     <VbCard>
       Current Page
       <input
-        type="number"
         v-model.number="activePage"
+        aria-hidden="true"
+        type="number"
       />
     </VbCard>
     <VbCard>
       <table class="table table-bordered">
         <tr>
           <th>Description</th>
-          <th>Va Pagination</th>
+          <th>VaPagination</th>
         </tr>
         <tr>
-          <td>Default Pagination</td>
+          <td>Default</td>
```

Comment:
<img width="268" alt="image" src="https://user-images.githubusercontent.com/5394573/189231576-1a3226c0-5514-4e01-93d7-2b2e8b2f1f8c.png">

default preset doesn't work too well as pagination default. primary/secondary make more sense for me.

For default it looks like 4 items are selected and one is disabled.

Reply:
Agree. Will remove it from examples and demos.